inference-engine/src/gna_plugin/gna2_model_dump.hpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
file name may be confusing with other dumps (eg embedded)

Reply:
The whole code of this PR will be placed in the gna2_model_debug_log.cpp/.hpp.